code/modules/mob/living/carbon/human/species.dm

```diff
@@ -2038,6 +2040,19 @@ GLOBAL_LIST_EMPTY(roundstart_races)
 	if(flying_species) //species that already have flying traits should not work with this proc
 		return
 	flying_species = TRUE
+	if(wings_icons.len > 1)
+		var/list/wings = list()
+		for(var/W in wings_icons)
+			var/datum/sprite_accessory/S = GLOB.wings_list[W]	//Gets the datum for every wing this species has, then prompts user with a radial menu
+			var/image/img = image(icon = 'icons/mob/clothing/wings.dmi', icon_state = "m_wingsopen_[S.icon_state]_BEHIND")	//Process the HUD elements
+			img.transform *= 0.5
+			img.pixel_x = -32
+			wings += list(S.name = img)
+		wings_icon = show_radial_menu(H, H, wings, tooltips = TRUE)
+		if(!wings_icon)
+			wings_icon = wings_icons[rand(1, wings.len)]
```

Comment:
```suggestion
			wings_icon = pick(wings_icons)
```
Also have this happen if there's no client in `H` to select one, without having to create the radial menu.